schema.json

```diff
@@ -77,6 +77,39 @@
             "installSize": {
                 "type": "integer",
                 "minimum": 0
+            },
+            "start": {
+                "type": "object",
+                "required": [
+                    "locomotive"

```

Comment:
Why is only the "locomotive" a required field? On its own, that's not enough for a set of valid choices, I think.